java-manta-client/src/main/java/com/joyent/manta/client/crypto/AesCbcCipherDetails.java

```diff
@@ -22,24 +24,101 @@
     /**
      * Instance of AES128-CBC cipher.
      */
-    public static final AesCbcCipherDetails INSTANCE_128_BIT = new AesCbcCipherDetails(128);
+    private static volatile AesCbcCipherDetails instance128Bit;
```

Comment:
This change would break any consumer which uses `INSTANCE_{128,192,265}_BIT` and as such would be a breaking change.